datus-redshift/tests/integration/conftest.py

```
@@ -0,0 +1,230 @@
+    )
+
+
+@pytest.fixture
+def connector(config: RedshiftConfig) -> Generator[RedshiftConnector, None, None]:
+    """Create and cleanup Redshift connector for integration tests."""
+    conn = None
+    try:
+        conn = RedshiftConnector(config)
+        if not conn.test_connection():
+            pytest.skip("Database connection test failed")
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)